llvm/include/llvm/CodeGen/MachinePassRegistry.def

```diff
@@ -124,6 +122,8 @@ DUMMY_FUNCTION_PASS("cfguard-check", CFGuardCheckPass, ())
 DUMMY_FUNCTION_PASS("gc-info-printer", GCInfoPrinterPass, ())
 DUMMY_FUNCTION_PASS("select-optimize", SelectOptimizePass, ())
 DUMMY_FUNCTION_PASS("callbrprepare", CallBrPrepare, ())
+DUMMY_FUNCTION_PASS("expand-large-div-rem", ExpandLargeDivRemPass, ())
```

Comment:
What's the difference between FUNCTION_PASS and DUMMY_FUNCTION_PASS? Why do these need to move?

Reply:
Fixed ExpandLargeDivRem in #71022